pkg/arvo/sys/vane/eyre.hoon

```diff
@@ -200,13 +207,44 @@
   %+  ~(put by unacked)  rid
   ?:  (lte u.sus ack)  0
   (sub u.sus ack)
+::  +find-channel-mode: deduce requested mode from headers
+::
+++  find-channel-mode
+  |=  [met=method:http hes=header-list:http]
+  ^-  ?(%json %jam)
+  =+  ^-  [hed=@t jam=@t]
+    ?:  ?=(%'GET' met)  ['x-channel-format' 'application/x-urb-jam']
+    ['content-type' 'application/octet-stream']
```

Comment:
Can we use `application/x-urb-jam` for PUT as well? Or is that header hard to set in situations we care about?

Reply:
You're right, we can just use the jam mime type here.